# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/thebes/D3D11Checks.cpp
# Commit: 75b04de7e99c
# Full Hash: 75b04de7e99cab7bed5cc8b0f2f6d52ddf174bf3
# Author: Jean-Yves Avenard <jyavenard@mozilla.com>
# Date: 2019-05-25 09:39:13
# Regressor Bug: 1550422
# File Overlap Count: 1
# Description:
#   Bug 1550422 - P15. Move Skip and Once gfxPrefs to StaticPrefs. r=jrmuizel
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D31259
# ==============================================================================

diff -r 91c3acdb2454 -r 75b04de7e99c gfx/thebes/D3D11Checks.cpp
--- a/gfx/thebes/D3D11Checks.cpp	Fri May 24 11:32:20 2019 +0000
+++ b/gfx/thebes/D3D11Checks.cpp	Fri May 24 11:32:54 2019 +0000
@@ -7,7 +7,6 @@
 #include "DXVA2Manager.h"
 #include "gfxConfig.h"
 #include "GfxDriverInfo.h"
-#include "gfxPrefs.h"
 #include "gfxWindowsPlatform.h"
 #include "mozilla/RefPtr.h"
 #include "mozilla/StaticPrefs.h"
@@ -140,7 +139,7 @@
   if (checked) return result;
   checked = true;
 
-  if (gfxPrefs::Direct2DForceEnabled() ||
+  if (StaticPrefs::Direct2DForceEnabled() ||
       gfxConfig::IsForcedOnByUser(Feature::HW_COMPOSITING)) {
     result = true;
     return true;
@@ -201,7 +200,7 @@
     return false;
   }
 
-  if (gfxPrefs::Direct2DForceEnabled() ||
+  if (StaticPrefs::Direct2DForceEnabled() ||
       gfxConfig::IsForcedOnByUser(Feature::HW_COMPOSITING)) {
     return true;
   }
@@ -213,7 +212,7 @@
       gfxInfo->GetAdapterVendorID(vendorID);
       gfxInfo->GetAdapterVendorID2(vendorID2);
       if (vendorID.EqualsLiteral("0x8086") && vendorID2.IsEmpty()) {
-        if (!gfxPrefs::LayersAMDSwitchableGfxEnabled()) {
+        if (!StaticPrefs::LayersAMDSwitchableGfxEnabled()) {
           return false;
         }
         gfxCriticalError(CriticalLog::DefaultOptions(false))